tools/release-it

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+# Create a new tag release of the browser extension.
+#
+# The current workflow for a browser extension release is:
+#
+#  1. Update the 'hypothesis' package dependency to the latest version
+#  2. Update the package version to match the 'hypothesis' client package
+#     version
+#  3. Tag and push a new release of the browser-extension repo using this
+#     script
+#  4. When the Travis CI builds for the tag complete, upload the
+#     extension to the Chrome Web Store
+
+set -e
```

Comment:
Thanks. I didn't know about `set -u`. On my system at least, `set -eu` aborts the script rather than just warning about unset variables, which is good.